### DISCUSSION TITLE
Support for Capacitor v4

### DISCUSSION THREAD

**Discussion by DouglasCalora** [quasarframework/quasar#15602] [Ideas / Proposals]
It's been a while since capacitor v4 were released (8 months ago) the 5.0.0-alpha1 it's also released. There is a pull request opened adding support to v4, why not merged?
(https://github.com/quasarframework/quasar/pull/14226)

**Comment by felipejuzo02** [comment]
Up!!

**Comment by gcarpi** [comment]
Up!

**Comment by hammertime-red** [comment]
Up!

**Comment by metalsadman** [comment]
You can upgrade it manually while that PR is still pending. Should work fine following capacitor.js's upgrade guide.
